docs/characters/dendro/collei.md

```diff
@@ -174,6 +174,10 @@ import Constellation from '@site/src/components/char/Constellation'
 
 <TabItem value='c4' label='C4'>
 <Constellation char={char} constellation={4} />
+
+**Notes**
+
+* The buff persists for its full duration even when this character leaves party. Click [here](../../evidence/combat-mechanics/party-mechanics.md#debuffsteam-buffs-with-duration-persist-after-applier-leaves-party) for the finding. 
```

Comment:
Building is failing because there's no newline between the ending of the list and </TabItem>